x-pack/plugins/cases/common/types/api/configure/v1.test.ts

```diff
@@ -319,14 +321,61 @@ describe('configure', () => {
         right: { ...defaultRequest },
       });
     });
+
+    it('defaultValue fails if the type is not string', () => {
+      expect(
+        PathReporter.report(
+          TextCustomFieldConfigurationRt.decode({
+            ...defaultRequest,
+            default_value: false,
+          })
+        )[0]
+      ).toContain('Invalid value false supplied');
+    });
+
+    it('limits the default_value to ', () => {
```

Comment:
This test seems the same as the above. What is the difference?

Reply:
it s a mistake, ill remove 😬